streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/service/StreamCatalogService.java

```diff
@@ -22,11 +22,7 @@
 import com.google.common.base.Function;
 import com.google.common.base.Joiner;
 import com.google.common.base.Preconditions;
-import com.google.common.collect.Collections2;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Lists;
-import com.google.common.collect.Sets;
+import com.google.common.collect.*;
```

Comment:
I will revert the import wildcards, since they are contrary to Streamline team coding standards.  Thanks for informing me.